client-side/src/stories/header/header.jsx

```diff
@@ -86,10 +88,14 @@ function ResponsiveAppBar() {
         />
           </Box>
           <Box >
-            <Tooltip title="Open settings">
-              <IconButton onClick={handleOpenUserMenu} >
-              <Avatar>{getAvatarLetter()}</Avatar>
-              </IconButton>
+          <Tooltip title={currentname}>
+            { !currentProfile && < IconButton onClick={handleOpenUserMenu} >
+                <Avatar alt={currentname} src="/static/images/avatar/2.jpg" />
+              </IconButton>}
+        
+            { currentProfile && < IconButton onClick={handleOpenUserMenu} >
+                <Avatar alt={currentname} src="/static/images/avatar/2.jpg" />
+              </IconButton>}
```

Comment:
what is this duplication?
should the avatar be const? shouldn't it come from the user profile definition?